scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementSample.java

```diff
@@ -87,6 +90,11 @@ public void start(Stage stage) {
       surface.getElevationSources().add(elevationSource);
       scene.setBaseSurface(surface);
 
+      // add a scene layer
+      final String buildings = "http://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/layers/0";
+      ArcGISSceneLayer sceneLayer = new ArcGISSceneLayer(buildings);
```

Comment:
In samples, we generally don't define a string variable for URLs unless they are being re-used, so here you could just write this URL directly inside the method call:
```suggestion
      ArcGISSceneLayer sceneLayer = new ArcGISSceneLayer("http://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/layers/0");
```